app/controllers/two_factor_authentication/otp_expired_controller.rb

```diff
@@ -6,6 +6,9 @@ class OtpExpiredController < ApplicationController
 
     def show
       @otp_delivery_preference = otp_delivery_preference
+      @use_another_phone_path = use_another_phone_path
+      @unconfirmed_phone = unconfirmed_phone?
```

Comment:
If we use `@use_another_phone_path` like in the previous suggestion, we can remove this.

```suggestion
```